core/mutation/mutation.go

```diff
@@ -76,7 +77,15 @@ func (s *Server) GetMutations(ctx context.Context, in *tpb.GetMutationsRequest)
 	}
 
 	// Get highest and lowest sequence number.
-	highestSeq := uint64(resp.GetMapRoot().GetMetadata().GetHighestFullyCompletedSeq())
+	meta, err := internal.MetadataFromMapRoot(resp.GetMapRoot())
+	if err != nil {
+		return nil, grpc.Errorf(codes.Internal, err.Error())
```

Comment:
`grpc.Errorf` is deprecated.  Please use `status.Error` or `status.Errorf` if a format string is desired. 
Using `Errorf` without a format string makes me [nervous](https://en.wikipedia.org/wiki/Uncontrolled_format_string)

Reply:
Oops, that's a very good point.  Shall I fix grpc.Errorf usages throughout this file?